PowerShell/JumpCloud Commands Gallery/Mac Commands/Mac - Install CrowdStrike Falcon Agent.md

```diff
@@ -47,15 +95,15 @@ echo "Deleted /tmp/$TempFolder"
 
 #### Description
 
-This command will download and install the CrowdStrike Falcon Agent to the device if it isn't already installed.
+This command will download and install the CrowdStrike Falcon Agent to the device if it isn't already installed. The command will leverage CrowdStrike's API to find and download the latest version of the Falcon Agent onto the local machine.
 
 In order to use this command, follow the instructions from the [Installing the CrowdStrike Falcon Agent KB](https://support.jumpcloud.com/s/article/Installing-the-Crowdstrike-Falcon-Agent)
```

Comment:
We should credit the author of this workflow and say we based our bash implementation on Richard Purves' work https://richard-purves.com/2022/05/03/downloading-crowdstrike-via-api-for-fun-and-profit/

Reply:
Resolved with suggestion below!